tests/scripts/travis/install-extensions.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Please copy these files as they are in one commit and apply your changes in a second commit